### DISCUSSION TITLE
Fatal error: Trait 'Mpdf\PsrLogAwareTrait\MpdfPsrLogAwareTrait' not found in 0/src/Mpdf.php on line 28

### DISCUSSION THREAD

**Discussion by rravindranaik111** [mpdf/mpdf#1918] [Q&A]
### Guidelines

- [X] I understand that [if I fail to provide all required details, this issue may be closed without review](https://github.com/mpdf/mpdf/blob/development/.github/CONTRIBUTING.md).

### Description of the bug

While upgrading the mpdf we get following issue. Can anyone help us on the same.
Thanks In advance

Fatal error: Trait 'Mpdf\PsrLogAwareTrait\MpdfPsrLogAwareTrait' not found in /src/Mpdf.php on line 28
A PHP Error was encountered
Severity: Warning

```
Message: Cannot modify header information - headers already sent by (output started at system/core/Exceptions.php:271)

Filename: core/Common.php

Line Number: 570

Backtrace:

A PHP Error was encountered
Severity: Error

Message: Trait 'Mpdf\PsrLogAwareTrait\MpdfPsrLogAwareTrait' not found

Filename: src/Mpdf.php

Line Number: 28
```

### mPDF version

8.2

### PHP Version and environment (server type, cli provider etc., enclosing libraries and their respective versions)

7.4

### Reproducible PHP+CSS+HTML snippet suffering by the error

```
                require_once APPPATH.'/vendor/autoload.php';
		$mpdf = new \Mpdf\Mpdf();
		//$mpdf = $this->mpdf->load('','A4');		
		$mpdf->WriteHTML($html);
		$mpdf->Output($pdf_file_name,"D");	
```

**Comment by lpf260** [comment]
Please check your mPDF directory to see if it contains a psr-log-aware-trait in this directory.
`vendor->mpdf`
